apps/condo/domains/acquiring/constants/context.js

```diff
@@ -10,10 +10,20 @@ const CONTEXT_STATUSES = [
     CONTEXT_FINISHED_STATUS,
 ]
 
+const VAT_OPTIONS = ['0', '10', '20']
```

Comment:
This should be removed because of integration options, right?

Reply:
Yes. Removed.